models/user.py

```diff
@@ -20,7 +21,15 @@ def is_strong(password):
         :return: Is the password strong enough
         :rtype: bool
     '''
-    pass
+    if len(password) < 8 or \
```

Comment:
Very nice to use a regular expression! You'll have to explain me what they do! I'm very curious!